src/components/HiddenTaskList/HiddenTaskList.jsx

```diff
@@ -0,0 +1,89 @@
+const React = require('react')
+const { connect } = require('react-redux')
+
+const HiddenTaskListItem = require('../HiddenTaskListItem')
+const hookUpStickyNav = require('../hookUpStickyNav')
+const TaskVisibility = require('../../models/TaskVisibility')
+
+class HiddenTaskList extends React.Component {
+  onRestoreClick(event) {
+    event.currentTarget.blur() // defocus button
+    this.props.dispatch({ type: 'TASKS_RESTORE' })
+  }
+
+  cancel(event) {
+    event.preventDefault()
+    this.props.cancel()
+  }
+
+  navigation(tasks) {
+    if (tasks.length < 1) {
+      return
+    }
+    return (
+      <nav className="controls-container has-text-right">
+        <label className="label">With selected:</label>
+        <button
+          type="button"
+          onClick={e => this.onRestoreClick(e)}
+          className="control button is-link"
+          id="restore-button"
+          title="Restore selected"
+        >↩️</button>
+      </nav>
+    )
+  }
+
+  emptyListMessage(tasks) {
+    if (tasks.length > 0) {
+      return
+    }
+    return (
+      <p className="content">
+        There are no issues or pull requests that have been snoozed, archived,
+        or ignored for this filter.
+      </p>
+    )
+  }
+
+  render() {
+    const { activeFilter } = this.props
+    const hiddenTasks = this.props.tasks.filter(task => TaskVisibility.isHiddenTask(task))
+    return (
+      <div>
+        <nav id="hidden-task-list-navigation" className="top-nav nav">
+          <div className="nav-left">
+            <h1 className="title">
+              <a href="#" onClick={event => this.cancel(event)}>Tasks</a>
+              <span> / </span>
+              Hidden
+              <span className="subtitle"> in &ldquo;{activeFilter}&rdquo;</span>
```

Comment:
This is cool, but without a way of switching what your activeFilter is (other than going back, changing the filter, and returning), I feel like this could be annoying. Since the hidden tasks are ...hidden, I might forget what filter would reveal them, and have to go on a wild goose chase to find them so I could un-hide them. We may want to add an 'All' to the default filters list.

Any thoughts on making the hidden task list show all hidden tasks, rather than filtered?